packages/scandipwa/src/component/ProductActions/ProductActions.style.scss

```diff
@@ -217,8 +217,12 @@
         }
 
         button {
-            cursor: pointer;
-
+            &:not([disabled]) {
+                cursor: pointer;
+            }
+            &:disabled {
```

Comment:
Space between selectors please

Reply:
@AleksandrsKondratjevs what about adding stylelint to the project?

Reply:
Could be nice, depends if someone has time for it.

Reply:
>if someone has time for it.

Good remark :grinning: